src/gui/MixerView.cpp

```diff
@@ -545,49 +472,49 @@ void MixerView::keyPressEvent(QKeyEvent * e)
 	switch(e->key())
 	{
 		case Qt::Key_Delete:
-			deleteChannel(m_currentMixerLine->channelIndex());
+			deleteChannel(m_currentMixerChannel->channelIndex());
 			break;
 		case Qt::Key_Left:
-			if( e->modifiers() & Qt::AltModifier )
+			if (e->modifiers() & Qt::AltModifier)
 			{
-				moveChannelLeft( m_currentMixerLine->channelIndex() );
+				moveChannelLeft(m_currentMixerChannel->channelIndex());
 			}
 			else
 			{
 				// select channel to the left
-				setCurrentMixerLine( m_currentMixerLine->channelIndex()-1 );
+				setCurrentMixerChannel(m_currentMixerChannel->channelIndex() - 1);
 			}
 			break;
 		case Qt::Key_Right:
-			if( e->modifiers() & Qt::AltModifier )
+			if (e->modifiers() & Qt::AltModifier)
 			{
-				moveChannelRight( m_currentMixerLine->channelIndex() );
+				moveChannelRight(m_currentMixerChannel->channelIndex());
 			}
 			else
 			{
 				// select channel to the right
-				setCurrentMixerLine( m_currentMixerLine->channelIndex()+1 );
+				setCurrentMixerChannel(m_currentMixerChannel->channelIndex() + 1);
 			}
 			break;
 		case Qt::Key_Insert:
-			if ( e->modifiers() & Qt::ShiftModifier )
+			if (e->modifiers() & Qt::ShiftModifier)
 			{
 				addNewChannel();
 			}
 			break;
 		case Qt::Key_Enter:
 		case Qt::Key_Return:
 		case Qt::Key_F2:
-			renameChannel( m_currentMixerLine->channelIndex() );
+			renameChannel(m_currentMixerChannel->channelIndex());
 			break;
 	}
 }
 
 
 
-void MixerView::closeEvent( QCloseEvent * _ce )
+void MixerView::closeEvent(QCloseEvent * _ce)
```

Comment:
Extra space between type and `*` and remove `_` prefix from argument

Reply:
Fixed with commit 92878ce4030. Using reference/pointer at type like in other places in the code. Underscore was removed. Prefix removed.